guides/source/active_record_callbacks.md

```diff
@@ -397,14 +682,17 @@ Article destroyed
 Association Callbacks
 ---------------------
 
-Association callbacks are similar to normal callbacks, but they are triggered by events in the life cycle of a collection. There are four available association callbacks:
+Association callbacks are similar to normal callbacks, but they are triggered by
+events in the life cycle of a collection. There are four available association
+callbacks:
 
 * `before_add`
 * `after_add`
 * `before_remove`
 * `after_remove`
 
-You define association callbacks by adding options to the association declaration. For example:
+You define association callbacks by adding options to the association
+declaration. For example:
```

Comment:
Should we make the methods within the examples below private, like the other callback examples we've been showing? (since that's the general recommendation)

Reply:
✅